integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -106,10 +106,17 @@ public boolean supportsValuesForDynamicField(Class<?> javaType) {
 	}
 
 	@Override
-	public boolean supportsTotalHitsThreshold() {
+	public boolean supportsTotalHitsThresholdForSearch() {
 		return dialect.supportsSkipOrLimitingTotalHitCount();
 	}
 
+	@Override
+	public boolean supportsTotalHitsThresholdForScroll() {
+		// If we try to customize track_total_hits for a scroll, we get an error:
+		// "disabling [track_total_hits] is not allowed in a scroll context"
+		return false;
```

Comment:
Maybe we should document that if we use the ES backend, any `totalHitCountThreshold` defined on the query will be ignored by the scroll API.